src/IR.cpp

```diff
@@ -907,6 +907,46 @@ Stmt Atomic::make(const std::string &producer_name,
     return node;
 }
 
+namespace {
+
+const char *const instruction_op_names[] = {
+    // Shared:
```

Comment:
Shared -> Use by multiple backends

Reply:
Justify the overlap between this and intrinsics expressed using call nodes in a comment.

The important difference is that if you see one of these nodes, there's guaranteed to be an instruction on the target arch that implements it. No further lowering or emulation is required.

Reply:
Comments about comments should actually apply to header